doc/rst/source/supplements/seis/meca_notes.rst_

```diff
@@ -5,7 +5,7 @@ Bomford, G., Geodesy, 4th ed., Oxford University Press, 1980.
 
 Aki, K. and P. Richards, Quantitative Seismology, Freeman, 1980.
 
-\F. A. Dahlen and Jeroen Tromp, Theoretical Global Seismology, Princeton, 1998, p.167. Definition of scalar moment.
```

Comment:
We need the leading `\` to make rst happy. Otherwise we may get something like 

- A. Dahlen and Jeroen Tromp, Theoretical Global Seismology, Princeton, 1998, p.167. Definition of scalar moment.